### PR TITLE
Login view

### PR DESCRIPTION
**Pull request by mattlub**
relates #104 
- [x] sets up express handlebars
- [x] login view
- [x] use tachyons

**UPDATE: READY FOR REVIEW**

### CODE REVIEW

**Comment by mattlub** [comment]
@JWLD should be picking this up

**Comment by mattlub** [comment]
... I did it in the end... "styling" is super basic for now (but super reusable)

can be changed later lol

screen shot 2017-08-16 at 15 48 29

**Review by m4v15 — COMMENTED** [review]
@mattlub @des-des Looks great tbh, but a couple of things:

Do we need to make to dual language, cos it's not atm?

If we were planning on using the same style as Data Entry app we're not atm since I think they're planning on using BEM css (I cut out when we were talking about this the other day, sorry if I should have raised it then) @JWLD can confirm.

**Comment by mattlub** [comment]
@m4v15 
- dual language, I think so, but could be added after (what methods could we use to do it?).
- Jack did say Reem was planning to use BEM, but I figure we can change this after the data entry is done if we need to, or the data entry can just use these styles

Basically my idea was to get something ok merged since we are on a tight timescale.

**Comment by m4v15** [comment]
@mattlub 

Yeah sounds good, I just was making sure we were aware of those things.

**Review by m4v15 — APPROVED** [review]

File: src/tachyonsClasses.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (des-des) [inline src/tachyonsClasses.js]: If you want to reuse you tachyons classes, I would use a handlebars partial.

I think this is fine for now as styling will go away. Could we move this file out of root  though. `src/views` or something

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by mattlub** [comment]
@des-des I've made the changes :shipit:

**Review by des-des — APPROVED** [review]

**Pull request merged**